### DISCUSSION TITLE
Performance question on List conversion from python to c#.

### DISCUSSION THREAD

**Discussion by loneshark99** [pythonnet/pythonnet#2431] [Q&A]
I have a question with respect to performance.

I am trying to create a C# List object from a python list object with the list size of 96.  I do this over 18k times but this conversion causes lot of latency.  Is there any other way to convert a python list object of floats to C# List<Single> ?     The code in the red box causes most latency.  Any suggestions/pointers will be helpful. Thanks.

![image](https://github.com/user-attachments/assets/b136bff4-fe15-421c-98f2-f4bfe45e63ca)

Thanks Much!

**Comment by loneshark99** [comment]
@filmor Can you please provide some suggestion or pointer?

**Comment by LeeDongGeon1996** [comment]
@loneshark99 Hi
This might be helpful to you.
https://github.com/facioquo/stock-indicators-python/pull/207
